.semaphore/buckets.yml

```diff
@@ -0,0 +1,26 @@
+version: v1.0
```

Comment:
Add Semaphore2 support. Do you see anything that can't be in a public repo @silvexis?